modules/va_notify/app/sidekiq/va_notify/v2/send_email.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module VANotify
+  module V2
+    class SendEmail
```

Comment:
I know this was Nathan's suggestion, though there's 2 points I'd like to raise here: 

- Frequently, Sidekiq jobs are appended with `Job`...`SendEmailJob`
- While the end result is an attempted email sent, that's the happy path, and that's via a Delivery Service Provider, via the Notification-API. If we're transitioning to use this new job, perhaps we could prevent future issues with a name that reflects this nuance? `RequestEmailDeliveryJob` or `QueueEmailJob`? Food for thought...not a hard stop